moml/models/mgnn/djmgnn.py

```
@@ -401,30 +386,30 @@ def forward(self, x, edge_index, edge_attr=None, batch=None, dist=None, env_vec=
-                h_aggregated.size(1) if h_aggregated.dim() > 1 else self.graph_head[0].in_features,
-            ).to(h_aggregated.device if hasattr(h_aggregated, "device") else x.device)
-        else:
-            graph_pooled = self.pool(h_aggregated, current_batch)
-
-        if self.env_dim:
-            if env_vec is None:
-                env_vec = x.new_zeros(graph_pooled.size(0), self.env_dim)
-            env_emb = self.env_proj(env_vec) if self.env_proj else env_vec
-            graph_fused = torch.cat([graph_pooled, env_emb], 1)
-            h_aggregated = torch.cat([h_aggregated, env_emb], 1)
+            expected_graph_batch_size = current_batch.max().item() + 1 if current_batch.numel() > 0 else 0
```

Ruff (0.11.9)

Local variable `expected_graph_batch_size` is assigned to but never used

Remove assignment to unused variable `expected_graph_batch_size`

(F841)